src/encoded/static/components/search.js

```diff
@@ -502,39 +502,45 @@ globals.listingViews.register(Experiment, 'SingleCellUnit');
 globals.listingViews.register(Experiment, 'TransgenicEnhancerExperiment');
 
 
-const AnnotationComponent = ({ context: result, cartControls, mode, auditIndicators, auditDetail }, reactContext) => (
-    <li className={resultItemClass(result)}>
-        <div className="result-item">
-            <div className="result-item__data">
-                <a href={result['@id']} className="result-item__link">
-                    {datasetTypes[result['@type'][0]]}
-                    {result.description ? <span>{`: ${result.description}`}</span> : null}
-                </a>
-                <div className="result-item__data-row">
-                    <div><strong>Lab: </strong>{result.lab.title}</div>
-                    <div><strong>Project: </strong>{result.award.project}</div>
+const AnnotationComponent = ({ context: result, cartControls, mode, auditIndicators, auditDetail }, reactContext) => {
+    const targets = result.targets ? result.targets.map((target) => target.label) : [];
+    return (
+        <li className={resultItemClass(result)}>
+            <div className="result-item">
+                <div className="result-item__data">
+                    <a href={result['@id']} className="result-item__link">
+                        {datasetTypes[result['@type'][0]]}
+                        {result.description ? <span>{`: ${result.description}`}</span> : null}
+                    </a>
+                    <div className="result-item__data-row">
+                        <div><strong>Lab: </strong>{result.lab.title}</div>
```

Comment:
Using <strong> is fine. Albeit personally I tend to prefer putting style in SCSS as strong-tag is a relic from HTML of the 90's.

Reply:
Agreed. I wouldn’t do this anymore.